e2e/cypress/integration/system_console/ui_and_api/customization_spec.js

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Customization', () => {
+    let config;
+
+    before(() => {
+        // Get config
+        cy.apiGetConfig().then((response) => {
+            config = response.body;
+        });
+
+        // # Login as sysadmin and visit customization system console page
+        cy.apiLogin('sysadmin');
+        cy.visit('/admin_console/site_config/customization');
+    });
+
+    it('SC20330 - Can change Help Link setting', () => {
+        // * Verify that setting is visible and matches text content
+        const contents = ['The URL for the Help link on the Mattermost login page, sign-up pages, and Main Menu. If this field is empty, the Help link is hidden from users.'];
+        cy.get('label[for="SupportSettings.HelpLink"]').should('have.text', 'Help Link:').and('be.visible');
+
+        // * Verify that help setting is visible and matches text content
+        cy.get('label[for="SupportSettings.HelpLink"]').next().find('span').should('be.visible').and('have.text', contents[0]);
+
+        // * Verify the input box visible and has default value
+        cy.get('input[id="SupportSettings.HelpLink"]').should('have.value', config.SupportSettings.HelpLink).and('be.visible');
```

Comment:
Please add ``data-testid={this.props.id + 'input'}`` to input so this can be:
```suggestion
        cy.findByTestId('SupportSettings.HelpLinkinput').should('have.value', origConfig.SupportSettings.HelpLink).and('be.visible');
```